{{cookiecutter.app_name}}/{{cookiecutter.app_name}}/static/{% if cookiecutter.use_react == "yes" %}react{% endif %}/src/app/containers/dashboard/index.js

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+ import { AppProvider, Frame, ActionList, Card, TopBar, Navigation, Link, Button } from '@shopify/polaris';
+import { Icon } from 'octantis';
+import * as _ from 'lodash';
+import Popover from '../../components/popover/index.js';
+import Api from '../../api/index.js';
+
+import style from './style.css';
+
+const theme = {
+  colors: {
+    topBar: {
+      background: '#1e3d58',
+    },
+  },
+  logo: {
+    width: 140,
+    topBarSource:
```

Comment:
move this to styles.css or some relevant file

Reply:
1) Shopify gives provision to add images for logo in frame like this.
2) Frame doesn't accept 'className' props